handbook/fasse.md

```diff
@@ -194,40 +194,51 @@ data/
 
 ## Step 8. Scratch Space
 
-Scratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. The high-performance storage and resource isolation capabilities of a scratch space enhance performance, streamline workflows, and protect valuable data. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
+A acratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
 
-We have created a scratch space within our FASSE space: 
+**When to use a scratch space?** Whenever you need a temporary storage for intermediate data during processing or computations, use it to store data that is being actively manipulated or processed but is not needed for long-term storage.
 
+The lab has scratch spaces available within FASSE and CANNON: 
+
+FASSE path: 
+```
+/n/holyscratch01/LABS/dominici_nsaph/Lab 
```

Comment:
there is a projects subdirectory, this is where we want people to use scratch corresponding to their projects